source/docs/magento1/exposing-additional-attributes.md

```diff
@@ -0,0 +1,106 @@
+---
+id: exposing-additional-attributes
+title: Exposing additional attributes
+---
+
+In case where you need to expose new attributes on the existing endpoint you can simply add this attribute on your `api2.xml` file. For that, you need to retrieve resource name (API_UNIQUE_NAME value if you follow the basic structure shown in "Add custom endpoint section") and add your attribute(s) into the `attributes` node.
+
+- How to retrieve the resource name: you can retrieve this data on all `api2.xml` file.
+- Expose attribute `custom_attribute` for customer data:
```

Comment:
I wouldn't use this listing presentation and prefer some categories.

For instance here it's a section about adding `custom_attribute` for the most common resources.